python/sglang/test/test_utils.py

```diff
@@ -822,3 +822,61 @@ def run_one(_):
 def write_github_step_summary(content):
     with open(os.environ["GITHUB_STEP_SUMMARY"], "a") as f:
         f.write(content)
+
+
+def launch_server_in_nightly_test(
```

Comment:
Don't use this.

Reply:
May you update https://github.com/sgl-project/sglang/blob/main/test/srt/experiment_runner.py

Reply:
The old style of nightly evaluation is not good. Let's switch to a configurable approach to support any third-party tools. Something like this https://github.com/sgl-project/sglang/blob/main/test/srt/configs/sharegpt_config.yaml